assets/js/covid19.js

```diff
@@ -106,6 +106,18 @@
             }
             return false;
         });
+
+        var browserLanguage = navigator.language.slice(0, 2);
+        var language = ['de', 'en', 'es', 'fr', 'it'].indexOf(browserLanguage) > -1 ? browserLanguage : 'en';
```

Comment:
Can you also set the `lang` attribute of the `html` tag? That should cause the UI API to be localized as well. That might have to happen before the ally.ui.js script is invoked though.